### PR TITLE
Exporting Stackdriver events to Elastic Cloud

### PR DESCRIPTION
**Pull request by twenny**
This tutorial demonstrates how to stream Stackdriver events to Elastic Cloud.

- [x] - Editorial review - @ToddKopriva  
- [x] - Final production check - @ToddKopriva

### CODE REVIEW

**Comment by twenny** [comment]
I signed it!

**Comment by twenny** [comment]
Updated git email for CLA

**Comment by twenny** [comment]
Rerunning CI tests

**Comment by ToddKopriva** [comment]
I grabbed the review for this. 

@twenny , I'll get started on the review after you've updated the PR to resolve the conflicts in the index.js file.

**Comment by ToddKopriva** [comment]
@twenny :
Aaron, can you explain why your PR includes changes to two JavaScript files in the folder for another tutorial (tutorials/iot-device-to-device/)?

These changes are the only thing blocking the PR for your new tutorial from progressing.

**Comment by twenny** [comment]
@ToddKopriva Apologies - For some reason, Circle CI was showing a linting problem with that .js file, so I uh.. "fixed" it. I have resolved conflicts with the master version.

**Comment by ToddKopriva** [comment]
Thanks for the response. I'll get back to this PR soon (probably Monday).

**Comment by twenny** [comment]
No worries, and thanks for picking this up. Ready to resolve any issues you discover.

File: tutorials/exporting-stackdriver-elasticcloud/index.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+        gcloud iam service-accounts create logstash \
+            --display-name="Logstash to Stackdriver"
+
+    **Expected response:**
+
+        Created service account [logstash].
+    
+1.  Provide permissions to access Cloud Pub/Sub using the `pubsub.editor`
+    [predefined IAM role](https://cloud.google.com/pubsub/docs/access-control#tbl_roles). _Editor_ permissions are required
+    for version 1.1.0 of Logstash Pub/Sub plugin. This appears to have been
```
Comment (ToddKopriva) [inline tutorials/exporting-stackdriver-elasticcloud/index.md]: flagging this mention of version 1.1.0 as part of conversation in a later comment

File: tutorials/exporting-stackdriver-elasticcloud/index.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+    [addressed](https://github.com/logstash-plugins/logstash-input-google_pubsub/blob/master/lib/logstash/inputs/google_pubsub.rb#L211-L214) in version 1.2.0, which means _subscriber_
```
Comment (ToddKopriva) [inline tutorials/exporting-stackdriver-elasticcloud/index.md]: flagging this mention of version 1.2.0 as part of conversation in a later comment

File: tutorials/exporting-stackdriver-elasticcloud/index.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+1.  Create a Cloud Pub/Sub topic where Stackdriver will send events to be picked up by Logstash:
+
+        gcloud pubsub topics create stackdriver-topic
+
+    **Expected response:**
+
+        Created topic [projects/scalesec-dev/topics/stackdriver-topic].
+
+    Version 1.1.0 of the Logstash plugin will automatically create a subscription.
```
Comment (ToddKopriva) [inline tutorials/exporting-stackdriver-elasticcloud/index.md]: flagging this mention of version 1.1.0 as part of conversation in a later comment

File: tutorials/exporting-stackdriver-elasticcloud/index.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+## Install Logstash
+
+1.  Install logstash from Elastic.
+
+        wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add - && \
+        echo "deb https://artifacts.elastic.co/packages/6.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-6.x.list && \
+        sudo apt-get update && sudo apt-get install logstash
+
+1.  Install the Logstash Plugin for Cloud Pub/Sub. Version 1.2.0 of the Logstash Pub/Sub plugin is
```
Comment (ToddKopriva) [inline tutorials/exporting-stackdriver-elasticcloud/index.md]: I just checked, and it looks like version 1.2.1 is now available. I presume that that means that you need to update this section. I flagged a few instnace of references to specific versions above, too, just to make sure that you considered those.

File: tutorials/exporting-stackdriver-elasticcloud/index.md [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+
+    **Expected response:**  
+
+    (after many lines...)
+
+        Validating /home/aaron/logstash-input-google_pubsub/logstash-input-google_pubsub-1.1.0.gem
+        Installing logstash-input-google_pubsub
+        Installation successful
+
+    When this issue is resolved, it should be possible to replace the git clone/checkout process above with
```
Comment (ToddKopriva) [inline tutorials/exporting-stackdriver-elasticcloud/index.md]: This seems to be the last mention of specific versions of this plugin that may need to be updated.

**Comment by ToddKopriva** [comment]
@twenny , do you need anything more from me at this point?

**Comment by twenny** [comment]
Thank you @ToddKopriva, I will update this tutorial with the latest version info.

**Comment by ToddKopriva** [comment]
This tutorial has been published here:
https://cloud.google.com/community/tutorials/exporting-stackdriver-elasticcloud

Thanks again for contributing this.

**Comment by twenny** [comment]
Thank you for your patience and guidance, Todd.

**Pull request merged**